sdk/src/main/java/com/sygic/travel/sdk/Sdk.kt

```diff
@@ -34,4 +40,10 @@ class Sdk(clientId: String, xApiKey: String, context: Context) {
 		get() {
 			return kodein.instance()
 		}
+
+	fun reloadAuthorizationServerUrl(authServerUrl: String) {
```

Comment:
Probably correct solution will be to make it a constructor dependency. 
Also, for our usecase, we would create two Kodeins immidiately.

Reply:
Please, read the configuration urls from the object and then put them into kondein.